borrowing/serializers.py

```diff
@@ -0,0 +1,47 @@
+from rest_framework import serializers
+from .models import Borrowing
+
+
+class BorrowingSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Borrowing
+        fields = (
+            "id",
+            "borrow_date",
+            "expected_return_date",
+            "actual_return_date",
+            "book",
+            "user"
+        )
+
+
+class BorrowingListSerializer(BorrowingSerializer):
```

Comment:
u can use `serializers.ModelSerializer` if you define all variables, cause in future u can have conflict by inheritance, but if u use `serializers.ModelSerializer` u are independ on BorrowingSerializer

Reply:
Resolved